pkgs/development/web/deno/default.nix

```diff
@@ -12,26 +13,27 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "deno";
-  version = "1.33.3";
+  version = "1.34.0";
 
   src = fetchFromGitHub {
     owner = "denoland";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-E1Usmj4f3jvzllKIrItzOIe4kf0DRQWwobVGuN291Ys=";
+    hash = "sha256-B9YzJsnbohzQQQj2Ec1szh/aY2zoYFOoEXTShwOzMlE=";
   };
-  cargoHash = "sha256-8L9UrFQTpwG2Gw4V72NvqdqQicG0HPRdmNJPhkWqj4s=";
+  cargoHash = "sha256-rwN7pEb6bDLb7YrcfuWC9Y0m7b+50AeLKMKx7WLKYT8=";
 
-  postPatch = ''
-    # upstream uses lld on aarch64-darwin for faster builds
-    # within nix lld looks for CoreFoundation rather than CoreFoundation.tbd and fails
-    substituteInPlace .cargo/config.toml --replace '"-C", "link-arg=-fuse-ld=lld"' ""
-  '';
-
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs =
+    [ installShellFiles ] ++
+    lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ rustc.llvmPackages.lld ];
   buildInputs = lib.optionals stdenv.isDarwin (
     [ libiconv darwin.libobjc ] ++
-    (with darwin.apple_sdk.frameworks; [ Security CoreServices Metal Foundation QuartzCore ])
+    (
+      with darwin.apple_sdk.frameworks;
+      [ Security CoreServices Metal Foundation QuartzCore ] ++
+      # for lld
+      lib.optionals (stdenv.isAarch64) [ CoreFoundation ]
```

Comment:
Doesn't change the error:

```
ld64.lld: error: cannot open /nix/store/dnyxqchjna2y3lrwxnmg16nv6f992c3p-apple-framework-CoreFoundation-11.0.0/Library/Frameworks/CoreFoundation.framework/CoreFoundation: No such file or directory
          clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
You built it on aarch64-darwin? ofborg hasn't started yet

Reply:
darn. the only thing I've found in nixpkgs that looks semi promising is using lld during linux to darwin cross compile in the darlings test but that's not great.

I guess we'll have to drop it still